### PR TITLE
README reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
...
 
-A simple, free software weather app for Android. It gathers data from [OpenWeatherMap](https://openweathermap.org/), via their public API.
+A simple, copylefted libre software weather app for Android. It gathers data from [OpenWeatherMap](https://openweathermap.org/), via their public API.
 
 [<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/0d/Get_it_on_F-Droid.svg/200px-Get_it_on_F-Droid.svg.png" style="float: left; padding-right: 10px;">](https://f-droid.org/repository/browse/?fdid=cz.martykan.forecastie)
-[<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/c/cd/Get_it_on_Google_play.svg/500px-Get_it_on_Google_play.svg.png" style="float: left; padding-right: 10px;" height="70">](https://play.google.com/store/apps/details?id=com.casticalabs.forecastie)
+[<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/7/78/Google_Play_Store_badge_EN.svg/200px-Google_Play_Store_badge_EN.svg.png" style="float: left; padding-right: 10px;" height="70">](https://play.google.com/store/apps/details?id=com.casticalabs.forecastie)
```
Comment (robinpaulson) [inline README.md:9]: What is the significance of this change?

File: README.md [hunk 2]
```diff
@@ -3,10 +3,10 @@
 [![Release](https://img.shields.io/github/release/martykan/forecastie.png)](https://github.com/martykan/forecastie/releases)
 [![Circle CI](https://circleci.com/gh/martykan/forecastie/tree/master.png)](https://circleci.com/gh/martykan/forecastie/tree/master)
 
-A simple, free software weather app for Android. It gathers data from [OpenWeatherMap](https://openweathermap.org/), via their public API.
+A simple, copylefted libre software weather app for Android. It gathers data from [OpenWeatherMap](https://openweathermap.org/), via their public API.
```
Comment (robinpaulson) [inline README.md:6]: Yes, that's better, I agree.

File: README.md [hunk 3]
```diff
@@ -21,12 +21,12 @@ A simple, free software weather app for Android. It gathers data from [OpenWeath
 [<img src="https://i.imgur.com/olkk2cB.png" width=250>](https://i.imgur.com/olkk2cB.png)
 
 ## Contribution
-If you have any ideas or issues, don't hesitate to make contact via the Issues page. Every contribution is welcome. See [Contributing](https://github.com/martykan/forecastie/blob/master/CONTRIBUTING.md) for further information; please also read our [Code of conduct](https://github.com/martykan/forecastie/blob/master/CODE-OF-CONDUCT.md).
+If you have any ideas or issues, don't hesitate to make contact via the Issues page. Every contribution is welcome. See [Contributing](https://github.com/martykan/forecastie/blob/master/CONTRIBUTING.md) for further info.
```
Comment (robinpaulson) [inline README.md:24]: As in #451

File: README.md [hunk 4]
```diff
@@ -21,12 +21,12 @@ A simple, free software weather app for Android. It gathers data from [OpenWeath
...
 
 ## Licence
 [![GNU GPLv3 Image](https://www.gnu.org/graphics/gplv3-127x51.png)](http://www.gnu.org/licenses/gpl-3.0.en.html)  
 
-This application is Free Software: You can use, study share and improve it at your
+This app is libre software: You can use, study share and improve it at your
 will. Specifically you can redistribute and/or modify it under the terms of the
```
Comment (robinpaulson) [inline README.md:30]: I believe this was taken from gnu.org, as part of "how to apply the licence", so we should probably leave it as-is.

File: README.md (hunk 1 above)
Comment (comradekingu) [reply README.md:9]: That it works?

Comment (robinpaulson) [reply README.md:9]: A little more info, if you could?

Comment (comradekingu) [reply README.md:9]: It didn't work, now it does.
https://upload.wikimedia.org/wikipedia/commons/thumb/c/cd/Get_it_on_Google_play.svg/500px-Get_it_on_Google_play.svg.png
vs
https://upload.wikimedia.org/wikipedia/commons/thumb/7/78/Google_Play_Store_badge_EN.svg/200px-Google_Play_Store_badge_EN.svg.png

**Comment by robinpaulson** [comment]
Fixed manually.

**Pull request closed without merging**